bindings/tflite/java/com/google/iree/native/interpreter_wrapper.cc

```diff
@@ -0,0 +1,67 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "bindings/tflite/java/com/google/iree/native/interpreter_wrapper.h"
+
+#include "absl/memory/memory.h"
```

Comment:
drop this dep and just use `std::make_unique` (I'll send a PR shortly that does that for the rest of our code, to keep it consistent)

Reply:
https://github.com/google/iree/pull/5254